lib/pi_piper/driver.rb

```diff
@@ -0,0 +1,114 @@
+module PiPiper
+  class Driver
+    
+    VERSION = '0.1.0'
+    ALLOW = []
```

Comment:
This may be over-engineering. I don't think we need an explicit `ALLOW` constant but rather a set of public methods. Assuming I understand what this is for.